de_web_plugin.cpp

```diff
@@ -2593,22 +2599,27 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node)
                     else if (node->nodeDescriptor().manufacturerCode() == VENDOR_JENNIC &&
                              modelId == QLatin1String("lumi.sensor_wleak.aq1"))
                     {
+                        // TODO: change to fpStatusSensor/fpAlarmSensor
                         fpPresenceSensor.inClusters.push_back(IAS_ZONE_CLUSTER_ID);
                     }
                 }
                     break;
 
                 case POWER_CONFIGURATION_CLUSTER_ID:
                 {
-                    if (node->nodeDescriptor().manufacturerCode() == VENDOR_PHILIPS ||
-                        node->nodeDescriptor().manufacturerCode() == VENDOR_NYCE ||
-                        node->nodeDescriptor().manufacturerCode() == VENDOR_IKEA)
-                    {
+                    // @manup: is it safe to skip these tests?
```

Comment:
The tests were added because not all devices implement the power cluster properly (reporting doesn't work for example). For now I would like to keep them and extend the tests aka whitelist here.

Reply:
As I mentioned in #407, the seem to whitelisted in bindings.cpp as well.